alfresco-zipkin-repo/build.gradle

```diff
@@ -22,31 +22,6 @@ dependencies {
     testRuntime group: 'javax.servlet', name: 'servlet-api', version: '2.5'
 }
 
-amp {
-    from("$projectDir/src/main/resources"){
```

Comment:
You're not putting resources in the AMP anymore. Then please put the things that need to be in `src/main/amp`, where they will be picked up and placed in the amp automatically.
https://github.com/xenit-eu/alfresco-gradle-sdk#conventional-project-layout

Reply:
Restructured the resources, preserved identical file locations as the original amp